src/app/(main)/leaderboard/Table.tsx

```diff
@@ -30,18 +31,22 @@ export const TableCellWrapper = styled.div<{
 `;
 
 interface LeaderboardItem {
```

Comment:
Could we use supabase generated types here?

Reply:
imported